Dockerfile

```diff
@@ -67,15 +67,19 @@ ENV HOME /home/$PROJECTOR_USER_NAME
 ENV PROJECTOR_CONFIG_DIR $HOME/.config
 RUN set -ex \
     && microdnf install -y --nodocs \
-    shadow-utils wget git nss procps findutils which socat \
-    # Packages required by JetBrains products.
-    libsecret jq \
+    shadow-utils wget git nss procps findutils which socat jq \
     # Java 11 support
     java-11-openjdk-devel \
     # Python support
     python2 python39 \
     # Packages needed for AWT.
     libXext libXrender libXtst libXi libX11-xcb mesa-libgbm libdrm freetype \
+    # Install libsecret for the particular platform
+    && { [ $(uname -m) == "s390x" ] && yum install -y --nodocs \
+        https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm \
```

Comment:
probably yum need to be replaced by microdnf

Reply:
yes, if running in `ubi8-minimal` container, you don't have yum unless you explicitly install it first; `microdnf` works basically the same way but iirc doesn's support doing module rpm installs, only single rpms.